package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "request-progress",
-  "version": "2.0.1",
+  "version": "3.0.1",
```

Comment:
Do not bump the version, I will do it with [release-it](https://github.com/webpro/release-it)